src/v2/Apps/MyCollection/MyCollectionApp.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MyCollectionApp_me } from "v2/__generated__/MyCollectionApp_me.graphql"
+import { MyCollectionArtworksQueryRenderer } from "./Components/MyCollectionArtworks"
+
+interface MyCollectionAppProps {
+  me: MyCollectionApp_me
+}
+
+const MyCollectionApp: React.FC<MyCollectionAppProps> = ({ me }) => {
+  return <MyCollectionArtworksQueryRenderer />
```

Comment:
Why use a query renderer here?

Reply:
So yeah! all of this can be rendered using our normal flow (meaning, via server-side-rendering) and no need for initial loading states. 

If you take the fragment located in the `MyCollectionArtworksQueryRenderer` and hoist it up here into the `MyCollectionApp` file (and delete the query renderer file completely) things should just work.